src/Uno.UWP/Graphics/Display/DisplayInformation.macOS.cs

```diff
@@ -33,5 +35,25 @@ private void UpdateRawProperties()
 			ScreenWidthInRawPixels = (uint)(screenSize.Width * scale);
 			RawPixelsPerViewPixel = NSScreen.MainScreen.BackingScaleFactor;
 		}
+
+		/// <summary>
+		/// Sets the NativeOrientation property 		
+		/// </summary>
+		private void UpdateNativeOrientation()
+		{
+			NativeOrientation = DisplayOrientations.Landscape;
+		}
+
+		private void UpdateCurrentOrientation()
+		{
+			if (NSScreen.MainScreen.Frame.Width > NSScreen.MainScreen.Frame.Height)
```

Comment:
I think the orientation should be based on the window aspect ratio, not the screen itself.  What do you think?

Reply:
I think `DisplayInformation` depends on screen orientation only in this case

Reply:
Indeed, it should not use window size to detect the orientation.

Reply:
ok ok

Reply:
@MartinZikmund  @carldebilly 
Will ask since I got a bit confused with the comments regarding the orientation, is there anything I need to change here?

Reply:
No, I think you can keep it as it is 👍